src/style.css

```diff
@@ -1,3 +1,81 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
 body {
+  background-color: #000;
+}
+
+.header {
+  background-color: rgba(255, 255, 255, 1);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  box-shadow: 0 2 rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.logo {
+  margin-left: 50px;
+}
+
+.logo__img {
+  display: block;
+  height: 40px;
+}
+
+.nav__links {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link.last {
+  margin-right: 0 !important;
+}
+
+.nav__link {
+  position: relative;
+  color: #000;
+  display: inline-block;
+  line-height: 60px;
+  height: 60px;
+  margin: 0 20px 0 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
+  content: "";
+  width: 100%;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
   margin: 0;
+  padding: 0;
 }
+
+/* #pc {
+  color: #00acdc;
+} */
```

Comment:
Remove comments

Reply:
Everything seems to be fixed, please check